### ISSUE TITLE
Question: Unnamed Fieldset and HTML Spec vs WCAG

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/wcag#2140]
Hi All I am interesting in the opinion of other in the following question.

The Spec states 'Optionally a legend element, followed by flow content.' 

If a Fieldset grouping element is not named with a Legend, or other programmatic naming method, such as aria-labelledby. Is this a fail for WCAG, and if so, which success criterion 1.3.1 Info and Relationships or 2.4.6 Headings and Labels?

Reference

- [4.10.15 The fieldset element](https://html.spec.whatwg.org/#the-fieldset-element)

**Comment by patrickhlauke** [comment]
If there's no overarching label/legend visible, how would it fail 1.3.1? there's nothing conveyed through presentation that is not announced/conveyed in markup. and 2.4.6 doesn't mandate having headings or labels, only that if they ARE present, they must be descriptive.

**Comment by mraccess77** [comment]
A fieldset without a legend would convey a grouping in terms of a border but that should be communicated by the group role - although in practice that relationship may not be communicated to users of screen readers without an accessible name.

**Comment by JAWS-test** [comment]
Lack of legend can be a violation of SC 3.3.2 in certain circumstances, namely when a group label is necessary to understand the context of the form elements in the group, especially for radiobuttons and checkboxes

**Comment by LaurenceRLewis** [comment]
That answers my question. Closing
